docs/updating-ccid.md

```diff
@@ -0,0 +1,77 @@
+# Updating the CCID free software driver
+
+Note: This page is mainly intended for the Chrome OS Smart Card Connector
+project maintainers.
+
+## Background
+
+The Chrome OS Smart Card Connector app is shipped with a specific version of the
+CCID free software driver. This is a driver for smart card reader (not to be
+confused with drivers for smart cards, also sometimes called middleware).
+
+Reasons for pinning to a specific version of the driver:
+
+* potentially, adjustments in our code or in the building scripts might be
+  required after updating to new versions (note that we don't run the driver
+  verbatim, but instead port it under the Native Client technology for running
+  inside the JavaScript-based App);
+* we can make sure that the driver that we roll out as part of the App underwent
```

Comment:
nit: We (ditto)

Reply:
nit: Also, I think this sentence is more complicated than it needs to be. How about 'We can test the driver version (that we roll out) and assure its quality'? (The part in () is optional, I would not literally include ( or ) )